src/StoreProvider.lua

```diff
@@ -4,13 +4,16 @@ local storeKey = require(script.Parent.storeKey)
 
 local StoreProvider = Roact.Component:extend("StoreProvider")
 
-function StoreProvider:init(props)
+function StoreProvider.validateProps(props)
 	local store = props.store
-
 	if store == nil then
-		error("Error initializing StoreProvider. Expected a `store` prop to be a Rodux store.")
+		return false, "Error initializing StoreProvider. Expected a `store` prop to be a Rodux store."
 	end
+	return true
+end
 
+function StoreProvider:init(props)
+	local store = props.store
 	self._context[storeKey] = store
```

Comment:
```suggestion
	self._context[storeKey] = props.store
```
Small simplification. After this, LGTM!